swagger/v1/location_reference.json

```diff
@@ -1,5 +1,5 @@
 {
-  "LocationReference": {
+  "LocationReference1": {
```

Comment:
..and two LocationReference types? 🤔

Reply:
LocationReference1 refers to Location1, LocationReference2 refers to Location2 - to provide more realistic examples with distinct from_location and to_location

Reply:
Thankyou - makes sense 👍